WooCommerce/Resources/release_notes.txt

```diff
@@ -1,15 +1 @@
-- [*] Order editing: disable editing orders in other currencies, due to risk of showing incorrect information [https://github.com/woocommerce/woocommerce-ios/pull/14345]
-- [Internal] Update Zendesk SDK to 9.0.0 [https://github.com/woocommerce/woocommerce-ios/pull/14360]
-- [*] Payments > About Tap To Pay > Set Up Tap To Pay on iPhone > Try a Payment: it used to be no-op with a spinner, now it navigates to a test payment for testing Tap To Pay. [https://github.com/woocommerce/woocommerce-ios/pull/14340]
-- [*] Payments onboarding: for two onboarding errors that show a wp-admin Payments setup URL in a webview ("Payments plugin not setup" and "Stripe overdue" errors that can occur to both WooPayments and Stripe Extension), the webview is now set to Payments Connect instead of Payments Overview. [https://github.com/woocommerce/woocommerce-ios/pull/14297]
-- [*] Blaze: merchants can now select the text for the CTA on their campaigns [https://github.com/woocommerce/woocommerce-ios/issues/14355]
-- [*] Fixed: crash for missing NSMicrophoneUsageDescription [https://github.com/woocommerce/woocommerce-ios/pull/14386]
-- [***] Implemented support for the new Global Unique Identifier (GTIN) for products [https://github.com/woocommerce/woocommerce-ios/pull/14396]
-- [internal] Allow login for WPCOM suspended sites [https://github.com/woocommerce/woocommerce-ios/pull/14257]
-- [*] Menu tab > Payments: in the "deposits" summary view at the top, all mentions of "deposits/deposit/deposited" have been renamed to "payouts/payout/paid out" respectively to match the renaming in web. [https://github.com/woocommerce/woocommerce-ios/pull/14402]
-- [*] Receipts: Added message confirming receipt sent to customer after successful payment. [https://github.com/woocommerce/woocommerce-ios/pull/14390]
-- [internal] Simplifies App Icon by only using a single image for all supported platforms. [https://github.com/woocommerce/woocommerce-ios/pull/14429]
-- [*] Dashboard: Performance card now shows cached Visitors and Conversion data correctly. [https://github.com/woocommerce/woocommerce-ios/pull/14438]
-- [internal] Updated WooCommerceScreenshot to make it work again for generating screenshots. [https://github.com/woocommerce/woocommerce-ios/pull/14335]
-- [*] Jetpack Setup: Added support for WordPress.com account creation. [https://github.com/woocommerce/woocommerce-ios/pull/14471]
-
+Enhance your WooCommerce experience with this release! Enjoy GTIN support for products, improved Tap To Pay testing, and refined order editing. Merchants now have control over campaign call-to-action text in Blaze. Plus, improved receipt communication, a refined dashboard view, and WordPress.com account setup in Jetpack.
```

Comment:
```suggestion
It's two weeks since the last one, so we've got lots for you in this release. The app now supports GTIN global identifiers for products, and you can now update the call-to-action text in your Blaze campaigns. It's quicker to get your store set up for in-person payments, easier to see when a receipt has been sent for a payment, and we've fixed some bugs when trying out Tap to Pay on iPhone. Lastly, we've made some fixes to the stats in the dashboard.
```